modules/tinymce/src/core/test/ts/browser/newline/InsertNewLineTest.ts

```diff
@@ -162,4 +162,44 @@ describe('browser.tinymce.core.newline.InsertNewLineTest', () => {
     TinyAssertions.assertContent(editor, '<p><a href="#">a</a></p><p><a href="#"><img src="about:blank"></a></p>');
     TinyAssertions.assertSelection(editor, [ 1, 0 ], 0, [ 1, 0 ], 0);
   });
+
+  context('Blockquote', () => {
```

Comment:
We need additional tests for the new option types. If it set to say `div, blockquote` you should be able to split divs and blockquotes but not other container elements. If it's set to false it should not split anything. If it's set to true it should split any container block. This should probably be in it's own context maybe name it `end_container_on _empty_block` or something. There are some tests for this in the `EnterKeyTest.ts` but these are in a older legacy format so I think it's better that we write new tests for this here instead.